tests/forms_tests/tests/test_formsets.py

```diff
@@ -1632,27 +1632,27 @@ def test_repr(self):
         cases = [
             (
                 self.make_choiceformset(),
-                "<ChoiceFormSet: bound=False valid=Unknown total_forms=1>",
+                "<ChoiceSet: bound=False valid=Unknown total_forms=1>",
```

Comment:
I think this change is unexpected. On the ticket Tim commented.

> appending "Set" rather than "FormSet" if form.__name__ ends with Form

Therefore in this case where the name is 'Choice' we should still expect 'ChoiceFormSet'.

We should add a new test where the form is 'ChoiceForm' and see the Formset name change from 'ChoiceFormFormSet' to 'ChoiceFormSet'.

Reply:
Good point - @Antoliny0919 can you make this update and add another test case?

Reply:
@smithdc1, @sarahboyce 
I'm sorry, I think I misunderstood Tim's comment.
I'll try again !